packages/policies/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "@redhat-cloud-services/policies-client",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "doc": "typedoc --theme markdown --out doc --module commonjs --mode file --gitRevision master",
+    "build": "tsc",
+    "generate": "TS_POST_PROCESS_FILE='../../postProcess.sh' openapi-generator-cli generate -i $SPEC -g typescript-axios -o . --skip-validate-spec --enable-post-process-file",
+    "generate:prod": "SPEC='https://cloud.redhat.com/api/policies/v1/openapi.json' npm run generate"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/RedHatInsights/javascript-clients.git"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/RedHatInsights/javascript-clients/issues"
+  },
+  "homepage": "https://github.com/RedHatInsights/javascript-clients#readme",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {}
```

You should add `"axios": "^0.21.1"` in here. That way when the app installs this package they get the axios installed as well.
```suggestion
  "dependencies": {
    "axios": "^0.21.1"
  }
```